frontend/pages/shared-components/text-input/text-input.tsx

```diff
@@ -0,0 +1,49 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+import React, { FormEvent, useState } from 'react'
```

Comment:
follow up of a previous comment to see these shared-components outside of the pages/ directory